src/utils/transaction.ts

```diff
@@ -282,6 +283,51 @@ export const makeTxConfigFromTxReceipt = (
   return txConfig;
 };
 
+export const makeTxConfigFromRawTx = (
```

Comment:
I'm like 95% sure this exists somewhere already

Reply:
Yeah, I found and used a utility you had already made for token migration configs. Missed it at first.